src/main/java/com/minecolonies/coremod/colony/buildings/modules/GraveyardManagementModule.java

```diff
@@ -144,6 +144,23 @@ public GraveData getLastGraveData()
         return this.lastGraveData;
     }
 
+    /**
+     * Check if one of the citizens in the list is resting.
+     * @param citizens the citizens to check.
+     * @return true if so.
+     */
+    public boolean hasRestingCitizen(final Set<String> citizens)
+    {
+        for (final String citizen : citizens)
+        {
+            if (restingCitizen.contains(citizen))
+            {
+                return true;
+            }
+        }
+        return false;
+    }
+
```

Comment:
Why was this added?

Reply:
Was going to add the additional things to the PR doc =P